### DISCUSSION TITLE
try to use script example

### DISCUSSION THREAD

**Discussion by lk29** [arendst/Tasmota#19649] [Q&A]
tried the example berry script  

`>D

>W
bu(sw 'Light_on' 'Light_off')
ck(sw "Light on/off   ")
sl(0 100 dimmer "0" "Dimmer" "100")
sl(0 255 red "0" "red" "255")
sl(0 255 green "0" "green" "255")
sl(0 255 blue "0" "blue" "255")
sl(0 255 ww "0" "warm white" "255")
tx(color "color:   ")

`

result on screen
![Screenshot 2023-10-03 at 05-41-23 Tasmota - Main Menu](https://github.com/arendst/Tasmota/assets/12291632/34c9f416-a131-4057-b7d5-63d1a77c76c8)

why?

**Comment by sfromis** [comment]
That has nothing to do with [Berry](https://tasmota.github.io/docs/Berry/) but instead the "[scripting language](https://tasmota.github.io/docs/Scripting-Language/)". I'm not using that, but if what you copied here is the complete script, you're not setting up the variables correctly. Anyway, asking question is not what "Issues" is for, that would be in "Discussions" instead.

**Reply by lk29** [reply]
Ok

Yes, maybe I didn't configure it correctly.  

But I didn’t find a description of “how to configure it correctly.”  

If anyone knows how to do it correctly, please tell me.

**Reply by sfromis** [reply]
Just copying a small part from some example is not workable for learning about scripting. The learning curve is steeper than that, and it may take some time to grok the ideas about how things connect.

And the [example you copied from](https://tasmota.github.io/docs/Scripting-Language/#web-ui) has a lot of more content than what you seem to have cut it down to.

**Reply by lk29** [reply]
Sorry, I initially showed only part of the code. 

![IMG_20231003_214300.jpg](https://github.com/arendst/Tasmota/assets/12291632/4321ade9-3c8e-46e9-b3a5-03bfad0ce6db)

![IMG_20231003_214342.jpg](https://github.com/arendst/Tasmota/assets/12291632/66ca170f-9b41-4df7-9422-6a4a0bacef16)

Here is a simplified example code.  


It doesn't work as it should

**Reply by lk29** [reply]
I found the reason.  


it is necessary to use " 

![IMG_20231003_215457.jpg](https://github.com/arendst/Tasmota/assets/12291632/5eeb65e5-91a4-477c-93ed-19ce43fa4be1)

probably should be corrected on the site.

**Reply by lk29** [reply]
It’s strange that everything is correct on the website in the link, but where did I get the wrong example?
